Makefile

```
@@ -0,0 +1,118 @@
+RUST_TOOLCHAIN_NIGHTLY = nightly-2025-02-16
+SOLANA_CLI_VERSION = 3.0.0
```

This program's deps didn't get bumped yet.

```suggestion
SOLANA_CLI_VERSION = 2.3.4
```

---

I believe these configs can also come out of the manifest's `[workspace.metadata.cli]` section now, too?

---

Codama has a `toolchain` arg for its Rust renderer. You can probably hack it into the JS script to read from the Makefile. Maybe outside the scope of this PR though.